test/metabase/api/collection_test.clj

```diff
@@ -1479,39 +1454,9 @@
                         :entity_id       (:entity_id collection)
                         :color           "#ABCDEF"
                         :location        "/"
-                        :authority_level "official"
                         :parent_id       nil})
                       (mt/user-http-request :crowberto :put 200 (str "collection/" (u/the-id collection))
-                                            {:name "My Beautiful Collection", :color "#ABCDEF", :authority_level "official"})))))
-    (testing "Admins can edit the type"
```

Comment:
covered in `enterprise/backend/test/metabase_enterprise/content_management/api/collection_test.clj`